tests/testthat/test-runner.R

```diff
@@ -323,15 +323,13 @@ test_that("check_timeout kills timed out reports", {
   path <- orderly_git_example("demo")
   runner <- orderly_runner(path, workers = 2)
 
-  key1 <- runner$submit_task_report("minimal", timeout = 0)
-  key2 <- runner$submit_task_report("slow10", timeout = 0)
-  ## Sleep for a bit to allow "minimal" report to post TASK_COMPLETE message
-  Sys.sleep(5)
+  key <- runner$submit_task_report("slow10", timeout = 0)
+  Sys.sleep(2) ## Sleep to let runner pick up tasks
```

Comment:
would suggest a wait loop here until the task status has changed, but nbd really (just more reliable in practice and I think we have that from the runner?)